### PR TITLE
CAD-345 annotation constraints

### PR DESCRIPTION
**Pull request by CodiePP**
adaptations to upstream changes (PR https://github.com/input-output-hk/iohk-monitoring-framework/pull/533)

### CODE REVIEW

File: cardano-db-sync/src/Cardano/DbSync/Tracing/ToObjectOrphans.hs [hunk 1]
```diff
@@ -6,17 +6,14 @@
 
 module Cardano.DbSync.Tracing.ToObjectOrphans () where
 
-import           Data.Text (Text, pack)
+import           Data.Text (Text)
```
Comment (erikd) [inline cardano-db-sync/src/Cardano/DbSync/Tracing/ToObjectOrphans.hs]: Pack is such a generic name, and the rest of this project uses in qualified, so please switch this to use qualified. Eg:
````
import           Data.Text (Text)
import qualified Data.Text as Text
```
and then use `Text.pack` to match the code in the rest of this project.

Comment (CodiePP) [reply cardano-db-sync/src/Cardano/DbSync/Tracing/ToObjectOrphans.hs]: `pack` is not needed anymore. But, you are right, a qualified import of Text is preferable.

**Comment by CodiePP** [comment]
needs an updated cardano-node, so it's now a "WIP"

**Comment by erikd** [comment]
I recently add a couple of need typeclass instacnes in the `cardano-node`. We should make sure that not instances in this module shadow instances in `node`.

**Comment by CodiePP** [comment]
> I recently added a couple of needed typeclass instacnes in the `cardano-node`. We should make sure that no instances in this module shadows any instances in `node`.

yes, only one instance left here.

**Pull request merged**